Jenkinsfile.build

```diff
@@ -9,6 +9,9 @@ pipeline {
         choice(choices: ['test', 'prod'], description: 's3 bucket that the build will target', name: 'BUILD_DEST')
         choice(choices: ['default', 'test', 'prod'], description: 's3 bucket from which tiles are served', name: 'TILE_SOURCE')
     }
+        gitParameter(name: 'BRANCH_TAG',
```

Comment:
This block should be inside the `parameters` block.

Reply:
Good catch.